packages/main/src/plugin/kubernetes/contexts-states-dispatcher.ts

```diff
@@ -80,4 +83,12 @@ export class ContextsStatesDispatcher {
   getResourcesCount(): ResourceCount[] {
     return this.manager.getResourcesCount();
   }
+
+  updateResource(resourceName: string): void {
+    this.apiSender.send(`kubernetes-${resourceName}`);
```

Comment:
question: should it contain the word `update` in the event ?

`kubernetes-update-${resourceName}`